packages/manager/modules/pci/src/projects/project/storages/storages.module.js

```diff
@@ -23,11 +23,11 @@ angular
     'ui.router',
     'ovhManagerFilters',
     storagesComponents,
-    blocks,
     cloudArchive,
     databases,
     instanceBackups,
     objectStorage,
+    pciBlockStorage,
```

Comment:
not needed as a micro app is independent of other apps

Reply:
it's imported to allow feature flipping between uapp & monolith ; once the uapp is ok we will remove the imported module from pci